src/routes/courses/[id]/lessons/+page.svelte

```diff
@@ -186,16 +187,10 @@
 
                 <div class="flex items-center lg:mb-0">
                   {#if lessonEditing === lesson.id}
-                    <input
-                      type="date"
-                      name="lesson-date-picker"
-                      class="p-2 my-2 rounded-md sm:w-[179px]"
-                      value={formatDate(lesson.lesson_at)}
-                      on:input={(e) => (lesson.lesson_at = e.target.value)}
-                    />
+                    <DateField value={formatDate(lesson.lesson_at)} className="p-2 my-2 rounded-md sm:w-[179px] dark:bg-neutral-800 dark:text-white" label="" placeholder="" onChange={(e) => (lesson.lesson_at = e.target.value)}/>
```

Comment:
@R4Rohit23 You would need to use prettier to format the code.

Do you have prettier setup?